2021q2/libglvnd.adoc

```diff
@@ -0,0 +1,25 @@
+=== libglvnd on FreeBSD
+
+Contact: Kevin Bowling <kbowling@FreeBSD.org>
+
+Links: +
+link:https://gitlab.freedesktop.org/glvnd/libglvnd[libglvnd] URL: link:https://gitlab.freedesktop.org/glvnd/libglvnd[https://gitlab.freedesktop.org/glvnd/libglvnd]
+
+libglvnd (GL Vendor-Neutral Dispatch) "is a vendor-neutral dispatch layer for arbitrating OpenGL API calls
+between multiple vendors. It allows multiple drivers from different vendors to
+coexist on the same filesystem, and determines which vendor to dispatch each
```

Comment:
"in the same filesystem" maybe?

Also, I'd make it "which vendor's".

Reply:
> "in the same filesystem" maybe?

It's part of a quote from the project's README: https://gitlab.freedesktop.org/glvnd/libglvnd/-/blob/master/README.md

> Also, I'd make it "which vendor's".

Part of the same quote, but in any case `'s` doesn't make sense to me here. The original sentence is a bit complex but looks correct to me.